### PR TITLE
Pins sphinx to 1.6.5

### PR DESCRIPTION
**Pull request by dkliban**

### CODE REVIEW

File: ci/jjb/jobs/docs.yaml [hunk 1]
```diff
@@ -51,7 +51,8 @@
             # create a virtualenv in which to install packages needed to build docs
             virtualenv -p /usr/bin/python3 --system-site-packages ~/docs_ve
             source ~/docs_ve/bin/activate
-            pip3 install celery 'django<2' django-filter djangorestframework==3.6.4 djangorestframework-jwt drf-nested-routers psycopg2 sphinx git+https://github.com/snide/sphinx_rtd_theme.git@abfa98539a2bfc44198a9ca8c2f16efe84cc4d26 pyyaml virtualenv
+            pip3 install celery 'django<2' django-filter djangorestframework==3.6.4
```
Comment (dralley) [inline ci/jjb/jobs/docs.yaml]: Why the newline, and will this be problematic?

Comment (Ichimonji10) [reply ci/jjb/jobs/docs.yaml]: In editors that are set to not wrap code, this line of code is hard to read and edit as-is. I suspect a backslash is necessary, though.

File: ci/jjb/jobs/docs.yaml [hunk 2]
```diff
@@ -126,7 +127,8 @@
             # create a virtualenv in which to install packages needed to build docs
             python3 -m venv --system-site-packages ~/docs_ve
             source ~/docs_ve/bin/activate
-            pip3 install celery sphinxcontrib-swaggerdoc 'django<2' django-filter djangorestframework djangorestframework-jwt drf-nested-routers psycopg2 sphinx git+https://github.com/snide/sphinx_rtd_theme.git@abfa98539a2bfc44198a9ca8c2f16efe84cc4d26 pyyaml virtualenv
+            pip3 install celery sphinxcontrib-swaggerdoc 'django<2' django-filter
```
Comment (Ichimonji10) [inline ci/jjb/jobs/docs.yaml]: Please add a backslash here, too.

**Comment by werwty** [comment]
The docs builder issues were fixed via updating the docs. So we do not need to pin sphinx to 1.6.5

**Pull request closed without merging**